components/tokens/custom-spectrum/custom-vars.css

```diff
@@ -50,4 +50,5 @@ governing permissions and limitations under the License.
   /* static white / black background color for docs containers */
   --spectrum-docs-static-white-background-color: rgb(15, 121, 125);
   --spectrum-docs-static-black-background-color: rgb(206, 247, 243);
+  --spectrum-animation-distance: 6px;
```

Comment:
If this variable does not change based on scale or theme, you can just add this in the index.css file for overlay.